Registrar.Database/Migrations/M014CreateSettingsTableWithRegistrationDefault.cs

```diff
@@ -0,0 +1,32 @@
+﻿using FluentMigrator;
+
+namespace Registrar.Database.Migrations
+{
+    [Migration(14)]
+    public class M014CreateSettingsTableWithRegistrationDefault : Migration
+    {
+        public override void Up()
+        {
+            Create.Table("Settings")
+                .WithColumn("Id").AsInt32().Identity().NotNullable()
```

Comment:
Identity just specifies that it should be a key and auto increment. Drop the whole line and the system works the same. You can even put the PrimaryKey() part inline, and use it like Identity()